File: 1_A/VB_Net/HomeworkVB/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Me.RichTextBox1.Text = "Pong"
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Me.RichTextBox1.ResetText()


    End Sub

    Private Sub RichTextBox1_MouseEnter(sender As Object, e As EventArgs) Handles RichTextBox1.MouseEnter
        Me.RichTextBox1.BackColor = Color.Red

    End Sub

    Private Sub RichTextBox1_MouseLeave(sender As Object, e As EventArgs) Handles RichTextBox1.MouseLeave
        Me.RichTextBox1.BackColor = Color.White

    End Sub
End Class

File: Lesson1/1_A/CSharp/Homeworks/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Homeworks
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            this.richTextBox1.ResetText();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.richTextBox1.Text = "Pong";
        }

        private void richTextBox1_MouseEnter(object sender, EventArgs e)
        {
           this.richTextBox1.BackColor = Color.Red;
        }

        private void richTextBox1_MouseUp(object sender, MouseEventArgs e)
        {
           
        }

        private void richTextBox1_MouseHover(object sender, EventArgs e)
        {
           
        }

        private void richTextBox1_MouseLeave(object sender, EventArgs e)
        {
            this.richTextBox1.BackColor = Color.White;
        }
    }
}
